solidity/contracts/isms/multisig/AbstractMultisigIsm.sol

```diff
@@ -88,18 +107,16 @@ abstract contract AbstractMultisigIsm is IMultisigIsm {
      */
     function _verifyValidatorSignatures(
         bytes calldata _metadata,
-        bytes calldata _message
-    ) internal view returns (bool) {
-        (
-            address[] memory _validators,
-            uint8 _threshold
-        ) = validatorsAndThreshold(_message);
-        require(_threshold > 0, "No MultisigISM threshold present for message");
+        bytes calldata _message,
+        address[] memory _validators,
+        uint8 _threshold
+    ) internal pure returns (bool) {
         bytes32 _digest = CheckpointLib.digest(
             Message.origin(_message),
             MultisigIsmMetadata.originMailbox(_metadata),
             MultisigIsmMetadata.root(_metadata),
-            MultisigIsmMetadata.index(_metadata)
+            MultisigIsmMetadata.index(_metadata),
+            MultisigIsmMetadata.messageId(_metadata)
```

Comment:
How are we enforcing that the message ID that was signed matches the message that is being verified?